ocfweb/docs/docs/contact/irc.md

```diff
@@ -54,6 +54,52 @@ your email. To see if you are registered properly, try running `/msg NickServ
 info`. You should see your email address, and where you are logged in from,
 among other results.
 
+## ZNC
+
+[ZNC][znc] is an IRC network bouncer. It permanently
+connects to the IRC server, so users connecting through it can preserve
+their chat session. Most users don't need this, but IRC regulars may appreciate
+the extended chat history and the absence of join/leave messages.
+
+ZNC also integrates with NickServ, automatically authenticating for you with a
+nickserv module if you save a password with ZNC.
+
+### Creating a ZNC account
+
+Ask a root staffer (or ping #rebuild) to create a ZNC username / password.
+
+### Configuring ZNC
+
+Configuration is most easily done through the [ZNC web interface][webznc].
+It requires you to login using your staff-created ZNC account.
+
+Once you've logged in, under `Your Settings` you should set the following fields:
+
+* **Nickname**: your nickname
+* **Alt. Nickname**: your alternate nickname, if your primary is taken
+* **Ident**: should be same as nickname, used to uniquely identify you from
+  everyone else using IRC
+* **Networks**: add a network with server `irc.ocf.berkeley.edu +6697`,
+  **space included**.
+
+Click save at the bottom, and your ZNC account should be setup to connect to the
+main IRC server.
+
+### Connecting to ZNC
+
+The OCF ZNC server settings are:
+
+* **Server:** `irc.ocf.berkeley.edu`
+* **Port:** `4095` (requires SSL/TLS)
+
+You should also set your IRC client login settings:
+* **Use SSL [...]**: True
+* **Login method**: `Server password (/PASS password)`
+* **Password**: your ZNC password
```

Comment:
don't you have to do `user:password` for the password?

Reply:
Appears to be client-specific. I've logged in through HexChat and Mutter on iOS through ZNC port 4095 and neither needed user:password, but more traditional text-based clients may need it.

Reply:
Hmm, interesting. Might be worth mentioning because it seems you have to do user:password in ERC (which is what I use) and that caused me quite a lot of pain. You could say something like "(if this doesn't work try username:password)"

Reply:
Will do.